logging/google/cloud/logging/handlers/app_engine.py

```diff
@@ -65,9 +68,9 @@ def get_gae_resource(self):
         gae_resource = Resource(
             type='gae_app',
             labels={
-                'project_id': os.environ.get(_GAE_PROJECT_ENV),
-                'module_id': os.environ.get(_GAE_SERVICE_ENV),
-                'version_id': os.environ.get(_GAE_VERSION_ENV),
+                'project_id': self.project_id,
+                'module_id': os.environ.get(_GAE_SERVICE_ENV, ''),
```

Comment:
Do you mind going ahead and promoting `module_id` and `version_id` to public properties?

Reply:
Done :)